modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -785,7 +785,7 @@ protected function register_controls() {
 			],
 			'size_units' => [ 'px', 'em', 'rem' ],
 			'selectors' => [
-				'{{WRAPPER}}' => '--n-tabs-icon-size: {{SIZE}}{{UNIT}}',
+				'{{WRAPPER}}' => '--n-tabs-svg-icon-size: {{SIZE}}{{UNIT}}; --n-tabs-icon-size: {{SIZE}}{{UNIT}}',
```

Comment:
Please explain.

How is this related to fixing the vertical alignment?

Reply:
It isn't that task was updated, and this was added to this task.
fix the SVG icon is not visible in frontend when `inline font icons` experiment is on.

Reply:
Reverted to original code